tests/telefunc.ts

```diff
@@ -0,0 +1,12 @@
+import { runInRepo } from '../utils'
+import { RunOptions } from '../types'
+
+export async function test(options: RunOptions) {
+	await runInRepo({
+		...options,
+		repo: 'vikejs/telefunc',
+		beforeInstall: 'setup', // Needed for submodule initialization
+		build: 'build',
```

Comment:
@dominikg @brillout We can also make the build option array compatible so we can run multiple commands synchronously for the build step! like:
```
		build: ['setup', 'another command', 'build'],
```

Reply:
```
	build?: Task | Task[]
```

Reply:
I think it is good as beforeInstall, because this is needed before running `ni`. I would expect that every project will have a single command for build